### DISCUSSION TITLE
Raise PR for develop instead of master branch

### DISCUSSION THREAD

**Discussion by Splash34** [renovatebot/renovate#29690] [Request Help]
### What would you like help with?

None

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

36.55.0

### Please tell us more about your question or problem

Hi team,
renovate is working like a charm, the only problem i am stumbling about is, that I would like that the target branch of the PR is 'develop' instead of the 'master' branch.

Until now I have not found any property, maybe someone could assist. Also changing the default branch in Bitbucket did not work. 

Technical details:
-  baseBranches is set to: 'baseBranches': ['develop'],

Does anybody has a clue how to achieve this requirement?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Setting baseBranches=develop should work. But make sure you keep your renovate.json config in the default branch

**Comment by HonkingGoose** [comment]
> ### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.
> 
> 36.55.0

By the way, you're using a old version of Renovate. I recommend updating to a newer version.
